src/core/tests/type_prop/squeeze.cpp

```diff
@@ -17,178 +17,230 @@ using namespace std;
 using namespace ov;
 using namespace testing;
 
-TEST(type_prop, squeeze_axes_invalid_value) {
+namespace {
+
+template <typename T>
+class type_prop_local : public testing::Test {};
+
+using Squeeze_v0 = op::v0::Squeeze;
+using Squeeze_v15 = op::v15::Squeeze;
+
+using SqueezeTypes = ::testing::Types<Squeeze_v0, Squeeze_v15>;
+
+TYPED_TEST_SUITE(type_prop_local, SqueezeTypes);
+
+TYPED_TEST(type_prop_local, squeeze_axes_invalid_value) {
+    using Squeeze = TypeParam;
```

Comment:
It can be removed if test fixture is created similar like in [avg pool](https://github.com/praasz/openvino/blob/97b20c97ce08146e9be2329e0466153f9a93ee2a/src/core/tests/type_prop/avg_pool.cpp#L16)

Reply:
Done